_inc/client/security/index.jsx

```diff
@@ -83,6 +86,25 @@ export class Security extends Component {
 			return null;
 		}
 
+		const planClass = getPlanClass( get( this.props.sitePlan, [ 'product_slug' ] ) );
+		const activePurchaseProductSlugs = this.props.activeSitePurchases.map(
+			purchase => purchase.product_slug
+		);
+
+		const isPersonalPlan = 'is-personal-plan' === planClass;
+		const isFreePlanWithBackup =
+			'is-free-plan' === planClass &&
+			( activePurchaseProductSlugs.includes( 'jetpack_backup_daily' ) ||
```

Comment:
Should we also account for `jetpack_backup_daily_monthly`?

Reply:
This is now accounted for by using `planClass` instead of the plan slug directly.